sdk/core/src/main/java/com/klaviyo/coresdk/model/Profile.kt

```diff
@@ -35,17 +33,11 @@ class Profile(properties: Map<ProfileKey, Serializable>?) :
 
     internal fun setAnonymousId(anonymousId: String) = apply { this.anonymousId = anonymousId }
     internal var anonymousId: String?
-        get() = (this[ProfileKey.ANONYMOUS]) as String
+        get() = (this[ProfileKey.ANONYMOUS_ID]) as String
         set(value) {
-            this[ProfileKey.ANONYMOUS] = value
+            this[ProfileKey.ANONYMOUS_ID] = value
         }
 
-    // TODO internal?
-    fun addAppendProperty(key: String, value: Serializable) = apply {
```

Comment:
R.I.P.

Reply:
plus side, don't have to think about that TODO now